client/src/main/java/io/pravega/client/connection/impl/IoBuffer.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.connection.impl;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.pravega.common.MathHelpers;
+import io.pravega.shared.protocol.netty.WireCommands;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+
+class IoBuffer {
```

Comment:
Should this class be in package `common` along with all the other Byte-related classes?